scripts/constants.py

```diff
@@ -130,7 +130,7 @@ class ChainId(IntEnum):
     pass
 
 ETH_TOKEN_ADDRESS = 0x49D36570D4E46F48E99674BD3FCC84644DDD6B96F7C741B1562B82F9E004DC7
-DEPLOY_FEE = 100000
+DEPLOY_FEE = int(0.001 * 1e18)
```

Comment:
```suggestion
DEPLOY_FEE = int(1e15)
```

Reply:
actually I find it more readable in ETH, that's why I explictely wrote `value in ETH * 1e18`, wdty?